app/src/main/java/com/pysiak/simple/clock/ui/alarm/AlarmPresenter.java

```diff
@@ -0,0 +1,25 @@
+package com.pysiak.simple.clock.ui.alarm;
+
+/**
+ * Presenter for my Alarm fragment with {@link AlarmView}
+ *
+ * @author Yaroslav P.
+ */
+public class AlarmPresenter {
+
+    private AlarmView alarmView;
+    private AlarmFragment alarmFragment;
```

Comment:
Why AlarmFragment is here? It is incorrent. Remove it.

If you have copied this from SimpleClockPresenter - you need to leave it as Context.